File: codes/fizzbuzz.zig
const std = @import("std");

pub fn main() void {
    var i = 0; 
    while (i <= 100) : (i += 1) {
        if (i % 15) {
            std.debug.print("FizzBuzz");
        }
        else if (i % 3) {
            std.debug.print("Fizz");
        }
        else if (i % 5) {
            std.debug.print("Buzz");
        }
        else {
            std.debug.print("number={d}", .{i});
        }
    }
}

File: codes/fizzbuzz.pl
for ($i = 0; $i <= 100; $i++) {
    if ($i % 15 == 0) {
        print "FizzBuzz";
    } elsif ($i % 3 == 0) {
        print "Fizz";
    } elsif ($i % 5 == 0) {
        print "Buzz";
    } else {
        print $i;
    }
}